ansible/testbed_add_vm_topology.yml

```diff
@@ -130,3 +130,37 @@
 
   roles:
     - { role: eos, when: topology.VMs is defined and inventory_hostname in VM_targets } # role eos will be executed in any case, and when will evaluate with every task
```

Comment:
>when: topology.VMs is defined and inventory_hostname in VM_targets  [](http://example.com/codeflow?start=19&length=67)

add a condition 
{rols: eos, when role=eos and existing condition}
{role: sonic when role=sonic and existing condition}

Reply:
Done, please review it.